### PR TITLE
Update netflix_prep_install.sh

### PR DESCRIPTION
**Pull request by Teknoist**

### CODE REVIEW

File: netflix_prep_install.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ sudo apt-get update 2>&1 | dialog --title "Updating package database and install
 if [ ! -d /usr/share/doc/python-crypto ]; then
 	sudo apt-get install -q python-crypto
 fi
-sudo apt-get install -q -y build-essential python-pip libnss3 libnspr4
+sudo apt-get install -q -y build-essential python-pip libnss3 libnspr4 kodi-inputstream-adaptive
```
Comment (zjoasan) [inline netflix_prep_install.sh:13]: Not needed for OSMC since inputstream.adaptive is installed by default on OSMC, so i wont merge this

**Comment by Teknoist** [comment]
Can you add sudo echo ... part I forget first time add sudo

**Comment by dara-want-fuck-kurdistan** [comment]
Ok

**Comment by zjoasan** [comment]
Sorry for being late to the party, think i fixed the sudoers, never noticed it was needed for OSMC but now i do a if OSMC /bin/mount in sudoers then do nothing else add it.

Not 100% like your PR, but I wont add the kodi_inputstream_deb package, since it's not available in OSMC.  Closing PR, Thanks for your input

**Pull request closed without merging**